missions/templates/missions/ship_list.html

```diff
@@ -20,6 +20,7 @@ <h1>Ships</h1>
             <td>{{ ship.name }}</td>
             <td>
                 <a href="{% url 'ship_detail' ship.id %}">{% fa5_icon 'ship' 'fas' %}</a>
+                <a href="{% url 'default_assignment_list' ship.id %}">{% fa5_icon 'users' 'fas' %}</a>
                 <a class="text-danger" href="{% url 'ship_delete' ship.id %}">{% fa5_icon 'trash' 'fas' %}</a>
```

Comment:
Ich würde hier (und bei allen anderen Buttons) noch hilfe toggles einbinden. Siehe z.B. `data-toggle="tooltip" title="edit"` in registration_process.html

Reply:
macht Sinn

Reply:
fixed